flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/endpoint/hive/HiveServer2EndpointITCase.java

```diff
@@ -595,6 +598,27 @@ public void testGetInfo() throws Exception {
         }
     }
 
+    @Test
+    public void testUnrecognizedGetInfoTypeWontCauseFailure() throws Exception {
+        // When 'TGetInfoResp' is used and 'TGetInfoResp.infoValue' isn't set, hive service will
+        // close connection. Here using new 'TGetInfoType' CLI_ODBC_KEYWORDS added in hive3 sent by
+        // 'getSQLKeywords' to test whether it is handled appropriately.
+        Assumptions.assumeTrue(HiveVersionTestUtil.HIVE_310_OR_LATER);
+        try (Connection connection = ENDPOINT_EXTENSION.getConnection()) {
```

Comment:
I think it's better to use Thrift Client to send GetInfoRequest directly and we can also test in Hive2 environment.

Reply:
Applied.